cmd/erigon-cl/stages/stages_beacon_state.go

```diff
@@ -105,7 +105,12 @@ func SpawnStageBeaconState(cfg StageBeaconStateCfg, s *stagedsync.StageState, tx
 		if err := tx.ClearBucket(kv.BlockBody); err != nil {
 			return err
 		}
-		if err := tx.ClearBucket(kv.EthTx); err != nil {
+		ethTx := kv.EthTx
+		transactionsV3, _ := kvcfg.TransactionsV3.Enabled(tx)
+		if transactionsV3 {
+			ethTx = kv.EthTxV3
+		}
+		if err := tx.ClearBucket(ethTx); err != nil {
```

Comment:
`ethTx` variable name says it's "eth transaction". but it's actually table name. so, better give more descriptive variable name `ethTxTable` or something like this.

Reply:
yeah, I was also thinking that, will update with next prs then